internal/cmdutil/util.go

```diff
@@ -0,0 +1,29 @@
+package cmdutil
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+)
+
+func HomeDir() string {
```

Comment:
Would it make sense to use `UserCacheDir` [from stdlib](https://golang.org/src/os/file.go?s=11674:11709#L365)?

Reply:
I think it makes sense but I just moved this function from another package so I'm not sure if @ahmetb didn't use it on purpose or not.
btw, seems like it's using different env variable than `USERPROFILE` for windows.

Reply:
~/.kube is intentional. We have to keep using HOME to respect kubectl standards and backwards-compatibility.